packages/core/src/meta/__tests__/MetaManagerSpec.js

```diff
@@ -0,0 +1,49 @@
+import MetaManagerImpl from '../MetaManagerImpl';
```

Comment:
this test file has been removed in the target branch, should i also delete it?

Reply:
and tests remain in other file?